src/LibPQ.jl

```diff
@@ -7,15 +7,15 @@ using Compat.Dates
 using DocStringExtensions
 using Decimals
 using DataStreams
-using Base.Iterators: zip, product
-using IterTools: imap
+import Base.Iterators: zip, product
```

Comment:
Why change this? Since these functions aren't being overloaded, `using` is appropriate.

Reply:
I wonder that which one makes this warning on the ci.
```
┌ Warning: `using A: B` will only be allowed for single bindings, not modules. Use `using A.B` instead
│   caller = ip:0x0
└ @ Core :-1
```
https://ci.appveyor.com/project/invenia/libpq-jl/build/job/4d449u0l8khpd2il#L148

Reply:
oh, there's also Julia issue #25397 . I don't know much.